app/src/services/transformService.js

```diff
@@ -69,14 +69,19 @@ const transformService = {
 
     inspectionStatus: (obj) => {
       if (obj && !Array.isArray(obj)) {
-        return {...obj.dataValues};
+        let notes = [];
+        if (obj.Notes) {
+          notes = obj.Notes.map(c => { return {...c.dataValues}; });
```

Comment:
If you wanted to, this would be equivalent using a closure (instead of having to use an explicit return statement)
```js
notes = obj.Notes.map(c => ({ ...c.dataValues }));
```